Documentation/install/upgrade.rst

```diff
@@ -618,6 +623,77 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 New ConfigMap Options
 ~~~~~~~~~~~~~~~~~~~~~
 
+  * ``enable-remote-node-identity`` has been added to enable a new identity
+    for remote cluster nodes. This allows for network policies that distinguish
+    between connections from host networking pods or other processes on the local
+    Kubernetes worker node from those on remote worker nodes.  This is important
+    because Kubernetes Network Policy dictates that network connectivity from the
+    local host must always be allowed, even for pods that have a default deny rule for
+    ingress connectivity.   This is so that network liveness and readiness
+    probes from kubelet will not be dropped by network policy.  Prior to 1.7.x,
+    Cilium achieved this by always allowing ingress host network connectivity from any
+    host in the cluster.  With 1.7 and ``enable-remote-node-identity=true``, Cilium
+    will only automatically allow connectivity from the local node, thereby providing
+    a better default security posture.
+
+    The option is enabled by
+    default for new deployments when generated via Helm, in order to gain the benefits
+    of improved security. The option can be disabled
+    in order to maintain full compatibility with Cilium 1.6.x policy enforcement.
+    **Be aware** that upgrading a cluster to 1.7.x by using helm to generating a new Cilium config
```

Comment:
s/helm/Helm/ for consistency?
s/generating/generate/

Reply:
fixed "generating typo, thx.  

Capitalization suggestion for helm seems reasonable, but we seem to use lowercase helm elsewhere in the paragraphs (not just in the commands) so leaving as is for now and I figure if someone wants to change it we should probably change it more broadly in the docs.